test-validator/src/lib.rs

```diff
@@ -511,6 +520,13 @@ impl TestValidator {
             genesis_config.ticks_per_slot = ticks_per_slot;
         }
 
+        // Remove features tagged to deactivate
+        for deactivate_feature_pk in &config.deactivate_feature_list {
+            if FEATURE_NAMES.contains_key(deactivate_feature_pk) {
```

Comment:
Probably fine to just blindly remove the specified pubkeys. It'd be nice to log a warning when `remove()` returns `None` to raise typo errors though

Reply:
I see the potential that 
1. The provided Pubkey is not in FEATURE_NAMES
2. Provided is in FEATURE_NAMES but None is returned from `remove` (although that would imply an Account for a  Feature is None. Am I missing something as `create_genesis_config_with_leader_ex` fetches everything in the FEATURE_NAMES construct via `activate_all_features`